src/Controls/src/Core/ShadowTypeConverter.cs

```diff
@@ -0,0 +1,186 @@
+﻿#nullable disable
+
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Graphics.Converters;
+
+namespace Microsoft.Maui.Controls
+{
+	/// <summary>
+	/// Type converter for converting a properly formatted string to a Shadow.
+	/// </summary>
+	internal class ShadowTypeConverter : TypeConverter
```

Comment:
Make `ShadowTypeConverter` public and enabled nullable references `﻿#nullable enable`